specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/examples/GetCertificatePolicy-example.json

```diff
@@ -21,7 +21,11 @@
           "subject": "CN=KeyVaultTest",
           "ekus": [],
           "key_usage": [],
-          "validity_months": 297
+          "validity_months": 297,
+          "basic_constraints": {
+            "ca": false,
+            "description": "Optional for leaf certs but needs to be true for CA certificate, if building PKI on top of Key Vault."
```

Comment:
The `description` doesn't look right to me in here -- this is an example response, it wouldn't contain the description. 

The `description` should instead be added to the specification file `certificates.json`, shouldn't it?

Reply:
Additionally, the `basic_constraints` property would need to be added to the swagger. Even if this were valid example JSON, there's no representation in swagger.